.github/workflows/actions/entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+yum install -y udunits2-devel
+PYTHONS=("cp36-cp36m" "cp37-cp37m" "cp38-cp38" "cp39-cp39")
+
+cp /usr/include/udunits2/* /usr/include/
+
+SITECFG=cf_units/etc/site.cfg
+echo "[System]" > $SITECFG
+echo "udunits2_xml_path = /usr/share/udunits/udunits2.xml" >> $SITECFG
+
+for PYTHON in ${PYTHONS[@]}; do
+    /opt/python/${PYTHON}/bin/pip install --upgrade pip wheel setuptools setuptools_scm build twine auditwheel
+    /opt/python/${PYTHON}/bin/python -m build --sdist --wheel . --outdir /github/workspace/wheelhouse/
```

Comment:
You could factor `/github/workspace/wheelhouse/` into an environment variable, as it's also used in `line 17`.

Just a suggestion, not really fussed, your choice 👍

Reply:
Done!